graphql-jpa-query-annotations/src/main/java/com/introproventures/graphql/jpa/query/annotation/GraphQLCalcField.java

```diff
@@ -0,0 +1,13 @@
+package com.introproventures.graphql.jpa.query.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Target( { FIELD, METHOD })
+@Retention(RUNTIME)
+public @interface GraphQLCalcField {
```

Comment:
It would be better to use descriptive names such as `GraphQLCalculatedField`

Reply:
I think the name GraphQLCalculatedField is better.